### ISSUE TITLE
Scaling not yet clearly defined in the NMEA protocol

### ISSUE THREAD

**Issue by Simsys** [larus-breeze/documentation_and_utilities#5]
Max Betz wrote:
- The scaling is not yet clearly defined in the NMEA protocol. Do my assumptions fit here? 
Ballast (1.00 - 1.60) <= would it make sense to simply transfer a value in KG here?  What is the appropriate value range here (i.e. maximum ballast)?
- Bugs in % (0 - 30) <= In XCSOAR / Opensoar I find a setting between 0 ... 50 %.   (0 == clean, 50% wet surface "double sink rate") Which value range/scaling do you recommend here?
- Crew weight is still missing I think. Would 0 ... 300kg fit?  
- Switching between climb and pre-flight mode?  Our front end can currently be switched manually or use the information from the sensor. Should this be coupled with XCSOAR?

**Comment by Simsys** [comment]
Some information first: 
- The aircraft polar models in the front end and in XCSoar are similar but not the same. XCSoar, for example, does not recognize a pilot weight.
- The definition of the NMEA data sets was based on other existing connections. We cannot start from scratch here.
- There is currently no polar alignment between the front end and XCSoar. This is complex and currently out of scope.

In Detail:
- Ballast: The definition of ballast as a multiple of the usual take-off weight is sufficient. 
- Bugs in %: We can expand it to 50%, but i think this is not necessary
- Crew weight: is not known by xcsoar, so we can't exchange it
- Switching between climb and pre-flight: I don't see any point in defining a data set here, as XCSaoar switches this automatically and intervention here may be problematic.

To summarize. I see no need for action here at present. Our goal should be to get the defined functionality up and running. This already covers the most important points.

**Comment by Simsys** [comment]
Obviously there is no need for action here
